spec/classes/puppet_server_puppetserver_spec.rb

```diff
@@ -7,7 +7,9 @@
     context "on #{os}" do
       let (:default_facts) do
         os_facts.merge({
-          :puppetversion          => Puppet.version,
+          :puppetversion => Puppet.version,
+          :ipaddress     => '1.2.3.4',
```

Comment:
I think it's good a good practice to use [RFC 5735 example IPs](https://tools.ietf.org/html/rfc5737) or [RFC 1918 NAT IPs](https://tools.ietf.org/html/rfc1918).